ee/clickhouse/process_event.py

```diff
@@ -0,0 +1,181 @@
+import datetime
```

Comment:
Largely brought over from `tasks/process_event`. Should definitely make this more reusable